tests/blockchain/regtest/SendToPaymentCode.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <gtest/gtest.h>
+#include <chrono>
+#include <thread>
+
+#include "opentxs/OT.hpp"
+#include "ottest/data/crypto/PaymentCodeV3.hpp"
+#include "ottest/fixtures/blockchain/Regtest.hpp"
+#include "ottest/fixtures/blockchain/RegtestSimple.hpp"
+
+namespace ottest
+{
+
+TEST_F(Regtest_fixture_simple, send_to_payment_code)
+{
+    EXPECT_TRUE(Start());
+    EXPECT_TRUE(Connect());
+
+    const std::string name_alice = "Alice";
+    const std::string name_bob = "Bob";
+
+    Height begin = 0;
+    const auto blocks_number = 2;
+    auto coin_to_send = 100000;
+
+    auto [user_alice, success_alice] = CreateClient(
+        opentxs::Options{},
+        3,
+        core_wallet_.name_,
+        core_wallet_.words_,
+        address_);
+    EXPECT_TRUE(success_alice);
+
+    auto [user_bob, success_bob] = CreateClient(
+        opentxs::Options{},
+        4,
+        name_bob,
+        GetPaymentCodeVector3().bob_.words_,
+        address_);
+    EXPECT_TRUE(success_bob);
+
+    std::vector<std::reference_wrapper<const User>> users{user_alice, user_bob};
+    MineBlocksForUsers(users, target_height, blocks_number);
+    begin = target_height;
+    target_height += static_cast<int>(MaturationInterval()) + 1;
+
+    // mine MaturationInterval number block with
+    MineBlocks(begin, static_cast<int>(MaturationInterval()) + 1);
+    begin += MaturationInterval() + 1;
+
+    EXPECT_EQ(
+        GetBalance(user_alice),
+        amount_in_transaction_ * blocks_number * transaction_in_block_);
+
+    EXPECT_EQ(
+        GetBalance(user_bob),
+        amount_in_transaction_ * blocks_number * transaction_in_block_);
```

Comment:
this formula is being used 4 times in a row ... why not store it once and pass 4 times ?

Reply:
because when it fails you have reported error in common body function, and not directly from specific test number, it's harder to track.

Reply:
multiplication can fail ?

Reply:
Only expression you can extract, I thought you want to extract EXPECT* also :)